hack/update-codecgen.sh

```diff
@@ -144,7 +144,7 @@ for current in "${index[@]}"; do
   # We use '-d 1234' flag to have a deterministic output every time.
   # The constant was just randomly chosen.
   ###echo "DBG: running ${CODECGEN} -d 1234 -o ${base_generated_file} ${base_file}"
-  ${CODECGEN} -d 1234 -o "${base_generated_file}" "${base_file}"
+  ${CODECGEN} -d 1234 -o "${base_generated_file}" -nr 'ObjectMeta' "${base_file}"
```

Comment:
it is deterministic. But that's not the problem. The issue is that `ObjectMeta` (as a `api/v1` type) is processed first. All types embedding `ObjectMeta` get the promoted `CodecEncodeSelf` func and the ugorji generator skips the generation for that type.